xbmc/music/windows/GUIWindowMusicBase.cpp

```diff
@@ -706,7 +706,10 @@ void CGUIWindowMusicBase::UpdateButtons()
   msg2.SetLabel(g_localizeStrings.Get(744)); // Files
   g_windowManager.SendMessage(msg2);
 
-  msg2.SetLabel(g_localizeStrings.Get(15100)); // Library
+  msg2.SetLabel(g_localizeStrings.Get(14022)); // Library
```

Comment:
Done.

Looking at the code, 14022 is used in GUIWindowMusicPlaylistEditor.cpp, GUIWindowVideoBase.cpp, and 15100 was used only here. But strings.po says that 14022 is used in settings, and says nothing about 15100. 

Both messages have been in strings.po from the very beginning.Any idea about why?